template.md

```diff
@@ -34,8 +34,9 @@ Deadline for advancement eligibility: [**CALCULATE TIME HERE**](https://www.time
     1. Such proposals *should* include supporting materials when possible.
 1. Proposals looking to advance to stage 1 must be added (and noted as such) prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
     1. Such proposals *must* link to a proposal repository and they *should* link to supporting materials when possible.
-1. Proposals looking to advance to stages 2, 3, or 4 must be added (and noted as such) *along with the necessary materials* prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline.
-    1. Such proposals *must* link to supporting materials prior to the deadline, or else delegates may object to advancement solely on the basis of missing the deadline. If these materials change substantially after the deadline, the proposal may be disqualified from advancement, based on the committee’s judgment.
+1. Proposals looking to advance to stages 2, 3, or 4, as well as other normative changes to the standard or proposals in stage 3 looking to achieve consensus, must be added (and noted as such) *along with the necessary materials* prior to the deadline, or else delegates may withhold consensus solely on the basis of missing the deadline.
```

Comment:
```suggestion
1. Proposals looking to advance to stages 2, 3, or 4, as well as other normative changes to the standard or proposals in stage 3 looking to achieve consensus, must be added (and noted as such) *along with the necessary materials* prior to the deadline.
```

This is said below in the first bullet point. I think we don't need to repeat it?

Reply:
I had been interpreting it as two different things; here it says that if it's not on the agenda before the deadline, delegates may object, and in the first bullet point it says that if the materials aren't available before the deadline, delegates may also object.

It might make sense to fold the first bullet point into this main point, though.